base/operators.jl

```diff
@@ -168,6 +168,23 @@ isless(x::AbstractFloat, y::AbstractFloat) = (!isnan(x) & (isnan(y) | signless(x
 isless(x::Real,          y::AbstractFloat) = (!isnan(x) & (isnan(y) | signless(x, y))) | (x < y)
 isless(x::AbstractFloat, y::Real         ) = (!isnan(x) & (isnan(y) | signless(x, y))) | (x < y)
 
+"""
+    isgreater(x, y)
+
+Test whether `x` is greater than `y`, according to a fixed total order.
+`isgreater` is defined in terms of `isless`, but is not the opposite of that function.
+
+`isless` defines a fixed total order that ascends, while `isgreater` defines a
+fixed total order that descends. Both order values that are normally unordered,
+such as `NaN`, after all regular values and [`missing`](@ref) last. So for
+`isless` these values are biggest and for `isgreater` these values are
+smallest.
+
+# Implementation
+Types should not usually implement this function. Instead, implement `isless`.
+"""
+isgreater(x, y) = _is_reflexive_eq(x) && _is_reflexive_eq(y) ? isless(y, x) : isless(x, y)
```

Comment:
I wonder if it would be better to call this something else. I can see it's necessary for supporting the other functions below and have them match `min`, `max`, `minimum`, `maximum` - but it might be confusing if this were ever exported as not being the opposite of `isless`?

Reply:
Also perhaps it should be called `ismore` since the analogue of `isgreater` would be is `islesser`.

Reply:
I always read it like “is less than” and “is greater than” (regular comparatives) rather than “is the lesser of” and “is the greater of” (the rarer form of two-item-only comparisons like latter and former that I can’t remember the fancy grammar name of).

Nonetheless I do kind of like `ismore`.